client/state/data-layer/wpcom/sites/simple-payments/index.js

```diff
@@ -63,9 +63,15 @@ export function productToCustomPost( product ) {
 	return Object.keys( product ).reduce(
 		function( payload, current ) {
 			if ( metadataSchema[ current ] ) {
+				let value = product[ current ];
```

Comment:
it's part of the reducer iteration. In this case, `current` is the key of the `product` object, so `product[ current ]` is its value.

<img src="https://user-images.githubusercontent.com/77539/28532577-8ff2b09e-7070-11e7-966c-dd5ea0b46180.png" width="400px" />

<img src="https://user-images.githubusercontent.com/77539/28532582-956a7228-7070-11e7-9b46-08d2c081c37e.png" width="400px" />